packages/instant/src/util/analytics.ts

```diff
@@ -0,0 +1,35 @@
+import { ObjectMap } from '@0x/types';
+import { logUtils } from '@0x/utils';
+
+import { ANALYTICS_ENABLED } from '../constants';
+
+import { HeapAnalytics, heapUtil } from './heap';
+
+const evaluteHeapCall = (heapFunctionCall: (heap: HeapAnalytics) => void): void => {
+    if (!ANALYTICS_ENABLED) {
```

Comment:
per convo w/ @BMillman19 we should also guard against our heap being overwritten.  I could do this by checking the `appId` on the window's heap object